--- emacs/lisp/loadup.el
+++ emacs-master/lisp/loadup.el
@@ -605,7 +605,8 @@ comp-subr-arities-h
                            (if (eq system-type 'ms-dos)
                                "b-emacs.pdmp"
                              "bootstrap-emacs.pdmp"))
-                          (t (error "Unrecognized dump mode %s" dump-mode)))))
+                          (t (error "Unrecognized dump mode %s" dump-mode))))
+            (exec-path nil))
         (when (and (featurep 'native-compile)
                    (equal dump-mode "pdump"))
           ;; Don't enable this before bootstrap is completed, as the
-- 
2.51.0

